metamorph/server.go

```diff
@@ -309,7 +309,27 @@ func (s *Server) PutTransactions(ctx context.Context, req *metamorph_api.Transac
 	return resp, nil
 }
 
-func (s *Server) processTransaction(ctx context.Context, waitForStatus metamorph_api.Status, data *store.StoreData, TxID string, latestStatus metamorph_api.Status) *metamorph_api.TransactionStatus {
+func hasWaitForStatusReached(status metamorph_api.Status, waitForStatus metamorph_api.Status) bool {
```

Comment:
I don't think those statuses are correct. We have new orphan status with 10.

Reply:
These values do not reflect the values of the metamorph api in any way. This function merely determines at which status the response can be returned to the user for that particular tx.
Does it make sense?